### ISSUE TITLE
Problem with mochawesome: TypeError: Cannot read properties of undefined (reading 'length')

### ISSUE THREAD

**Issue by quad5** [LironEr/cypress-mochawesome-reporter#66]
### Environment

```markdown
Versions that don't work.
- OS: Mac 11.6
- Node:  16.5.0
- cypress-mochawesome-reporter: 3.0.0
- cypress: 9.7.2
- mochawesome: 7.1.2

Versions that works
- OS: Mac 11.6
- Node: 16.14.0
- cypress-mochawesome-reporter: 2.3.0
- cypress: 9.7.2
- mochawesome: 6.3.1
```

### What happened?

Please see the issue I logged with [mochawesome](https://github.com/adamgruber/mochawesome/issues/373). I am not sure who is the culprit.

### Config file

```json
The config file below is the working versions one. See Environment section for non-working versions.

{
  "name": "automation-testing",
  "version": "1.0.0",
  "description": "Automation Testing using js",
  "main": "index.js",
  "scripts": {
    "test": "cypress"
  },
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "@babel/plugin-proposal-private-methods": "^7.12.1",
    "cypress": "9.5.2",
    "cypress-fail-fast": "3.4.1"
  },
  "dependencies": {
    "@babel/core": "7.11.6",
    "chai": "4.2.0",
    "chai-string": "^1.5.0",
    "cypress-file-upload": "5.0.5",
    "cypress-localstorage-commands": "1.7.0",
    "cypress-mochawesome-reporter": "2.3.0",
    "cypress-terminal-report": "^2.2.0",
    "deepmerge": "^4.2.2",
    "lodash": "^4.17.21",
    "lodash.clonedeep": "4.5.0",
    "mocha": "9.2.2",
    "mochawesome": "6.3.1",
    "path": "^0.12.7",
    "uuid": "^8.3.0"
  }
}
```

**Comment by Idleness76** [comment]
@LironEr I also commented on the bug linked above.  This seems to occur if there are spec files that are either skipped, or otherwise don't have any tests present.

**Comment by LironEr** [comment]
Hey, sorry for the late response.

Is the issue still happening after upgrading mochawesome? https://github.com/adamgruber/mochawesome/issues/373

I will release a new version (`3.0.1`) with the updated mochawesome in the next hour

**Comment by Idleness76** [comment]
@LironEr this was fixed with moachawesome 7.1.3 and should be fine to include the latest version.
